templates/blocks/custom_video_block.html

```
@@ -0,0 +1 @@
+{{ self.video|safe }}
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)